### ISSUE TITLE
Petition to remove smoke bomb/bomb

### ISSUE THREAD

**Issue by cs6413110** [cs6413110/Pixel-Tanks#197]
Both useless. 0% use them, except flats for the one shot with one key for close range.

Time complexity for generating an idea: 10.0+8ex the required time to code it.
Time complexity for creating an image: 10.0+4ex the required time to code it.

12 items, minus these two, is 10. I think 10 items is completely fine. 

BUT:
We have to make a new item tab and.... uh... yeah... like that's ever happening.

**Comment by LostKing73** [comment]
yes, bcuz smoke is stupid and annoying

**Comment by BreadSpammer** [comment]
smoke bomb buff: if ppl r standing in it, blind them :D. maybe have a U.I thing where if smoke = true, then load an IMG. I don't think it should be full blindness, bc that would be annoying. I'm thinking more like blindness from Minecraft where u cant see far away.

just add Gernade and yeet bomb bc Evan doesn't feel like existing

**Comment by BreadSpammer** [comment]
grenade isn't even hard to code. its literally like dyanmite, but instead of sticking to walls, it bounces, instead of having a controlled detonation, its a timer, and instead of a constant speed, it starts fast, and slows down over time (imagine rolling a ball across grass)

**Comment by cs6413110** [comment]
> smoke bomb buff: if ppl r standing in it, blind them :D. maybe have a U.I thing where if smoke = true, then load an IMG. I don't think it should be full blindness, bc that would be annoying. I'm thinking more like blindness from Minecraft where u cant see far away.
> 
> just add Gernade and yeet bomb bc Evan doesn't feel like existing

You've made the fatal error of forgetting that it takes months to make an image. The complexity of an image like Minecraft blindness would kill a bread. We don't need grenade because it's just a dynamite rip off. In fact, breacher is effectively a better dynamite. The lack of unique items tells me we have too many already.

**Comment by cs6413110** [comment]
> grenade isn't even hard to code. its literally like dyanmite, but instead of sticking to walls, it bounces, instead of having a controlled detonation, its a timer, and instead of a constant speed, it starts fast, and slows down over time (imagine rolling a ball across grass)

The coding difficulty of making bouncing isn't worth it.

**Comment by LostKing73** [comment]
remove both. not wrth the time its taking cuz yall (bread, evan) r lazy

**Comment by BreadSpammer** [comment]
unless u want low quality :)

**Comment by LostKing73** [comment]
i want none of the above. remove.

**Comment by cs6413110** [comment]
No, I'm tired of people putting forth 2% 5 minute effort menus every time I crack down on inefficiency. We aren't doing blindness. We didn't vote on it. You spontaneously came up with it not 30 seconds ago.

**Comment by BreadSpammer** [comment]
bruh don't remove my smoke bomb

**Comment by BreadSpammer** [comment]
also u just don't like the items bc they were never balanced like the rest

**Comment by cs6413110** [comment]
> bruh don't remove my smoke bomb

You don't even use it.



> also u just don't like the items bc they were never balanced like the rest

No, it's because they aren't viable. Visibility doesn't matter. Bomb is just a mere utility against something that just shooting would be more effective. Even the gpas don't use them.

**Comment by BreadSpammer** [comment]
shut. just leave them in the game? it saves on not having to deal with broken accounts, and don't need to make a menu

**Comment by 3foe** [comment]
nu uh remove

**Comment by cs6413110** [comment]
> shut. just leave them in the game? it saves on not having to deal with broken accounts, and don't need to make a menu

Lazy.

**Comment by BreadSpammer** [comment]
dONT remOVE SmoOKEE :(

**Comment by CelestialKnight7** [comment]
I'm making the friking breacher img when I don't have 30 assignments to do

also no one reads final balancing. I already explained how to fix smoke, just make 5 smoke images that only play the decay animation when they die. could also increase the size of them so u cant see thru any cracks. this would solve the lag problem but as for the use problem u can make it so that people on the team that placed it can see thru it at like 50%transparency so its good for the person placing it and would discourage the enemy from going in there cuz it could be nothing or it could be 10 turrets. also you could increase the duration to be as long as, or almost as long as  the cool down u so u could have constant cover on an area.

**Comment by CelestialKnight7** [comment]
breacher die:
👍 murder
👎 life

**Comment by CelestialKnight7** [comment]
@cs6413110 @BreadSpammer @DarkMemeGod411 @LostKing73 @CelestialKnight7

**Comment by LostKing73** [comment]
you almost got me by swapping the emojiis

**Comment by cs6413110** [comment]
> ur just killing item diversity for no reason. like just live and let live. also removing the item slots will make it harder to replace them later

The reasons are:
- Degrades performance with 1600+ explosions streaming 60 updates/sec for 10 seconds
- No one uses it
- Low quality image design
- Very low application in fights

**Comment by CelestialKnight7** [comment]
> > ur just killing item diversity for no reason. like just live and let live. also removing the item slots will make it harder to replace them later
> 
> The reasons are:
> 
> * Degrades performance with 1600+ explosions streaming 60 updates/sec for 10 seconds
> * No one uses it
> * Low quality image design
> * Very low application in fights

also no one reads final balancing. I already explained how to fix smoke, just make 5 smoke images that only play the decay animation when they die. could also increase the size of them so u cant see thru any cracks. this would solve the lag problem but as for the use problem u can make it so that people on the team that placed it can see thru it at like 50%transparency so its good for the person placing it and would discourage the enemy from going in there cuz it could be nothing or it could be 10 turrets. also you could increase the duration to be as long as, or almost as long as the cool down u so u could have constant cover on an area.

**Comment by CelestialKnight7** [comment]
1. I explained how to reduce explions
2. cuz it needs buff/rework
3. :(
4. cuz it needs buff/rework

**Comment by CelestialKnight7** [comment]
@LostKing73 @cs6413110 what do u have against breacher

**Comment by cs6413110** [comment]
@CelestialKnight7 
Ok, that solves the performance issue but I don't think you understand the problem behind the mechanic of visibility in the first place. Taking away visibility is not good for any game because it basically stops pvp. If I can't see, I can't play the game. Smoke bomb, even with a larger, more performant explosion, will lead to a whole toxic play style that just interrupts combat or hides. The only real benefit is for airstrike and turret, but with the airstrike buff the size of the airstrike is bigger than the cloud. And turrets need nerfs, not buffs. It just doesn't add enough to the game for it to be worth it.

**Comment by LostKing73** [comment]
^^^

**Comment by LostKing73** [comment]
> @LostKing73 @cs6413110 what do u have against breacher

too much work for a mid situational item

**Comment by CelestialKnight7** [comment]
> > @LostKing73 @cs6413110 what do u have against breacher
> 
> too much work for a mid situational item

do u remember the original issue, its basically portable stealth, it can break invincible block can do big damage to anyone trying to hide behind a wall, can be stuck to a person for that 100 damage while also having other applications

**Comment by CelestialKnight7** [comment]
and by portable stealth I just mean the breaking invincible block part

**Comment by BreadSpammer** [comment]
ye, and if u want smoke DED so much, then what will replace it??? it either stays or u come up with a better idea. *cough* grenade *cough*

**Comment by cs6413110** [comment]
> @LostKing73 @cs6413110 what do u have against breacher

- Duplicate of dynamite(100 damage to players) BUT with an upgrade where it can break blocks
- Requires a new image which will take too long(I know you're busy but we can't just keep on waiting)
- Requires an animation
- Non essential to gameplay, very case specific, making it bad for general purpose battles(except for the fact that it undermines dyna)

**Comment by BreadSpammer** [comment]
bruh I already have an image

**Comment by cs6413110** [comment]
> ye, and if u want smoke DED so much, then what will replace it??? it either stays or u come up with a better idea. _cough_ grenade _cough_

Read the initial issue. 10 items. Not 12.

**Comment by BreadSpammer** [comment]
imma dtap u after seminary

**Comment by CelestialKnight7** [comment]
bomb is worse than breacher tho, why keep bomb when u can replace it for something better.
if u think about it bomb is situational as well except we gave it so many applications cuz we thought It was bad that now its still situational so u barely use it but when u do it annoyes the other person cuz u can get out of any situation u want

**Comment by cs6413110** [comment]
Uh, we're removing both?

**Comment by cs6413110** [comment]
Me thinking remove smoke bomb won 3 - 2 but then I realized we got a flat vote.

**Comment by CelestialKnight7** [comment]
ye :(

**Comment by LostKing73** [comment]
> bomb is worse than breacher tho, why keep bomb when u can replace it for something better. if u think about it bomb is situational as well except we gave it so many applications cuz we thought It was bad that now its still situational so u barely use it but when u do it annoyes the other person cuz u can get out of any situation u want

were not having ANY

**Comment by BreadSpammer** [comment]
![pixil-frame-0 - 2023-12-21T100936 546](https://github.com/user-attachments/assets/ea622989-5b16-409d-8cad-4dec5dd88b71)
![pixil-frame-0 - 2023-12-21T095611 487](https://github.com/user-attachments/assets/8a6101e9-67b6-40a2-947a-908c447771b5)
placeholder imgs for breacher while Evan makes a proper image
